extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -289,11 +305,52 @@ public function processCuratedContent( $data ) {
 	private function processCuratedContentItem( $item ) {
 		if ( !empty( $item[ 'article_id' ] ) ) {
 			$title = Title::newFromID( $item[ 'article_id' ] );
+
 			if ( !empty( $title ) ) {
 				$item[ 'article_local_url' ] = $title->getLocalURL();
+
 				return $item;
 			}
 		}
+
 		return null;
 	}
+
+	/**
+	 * @desc To save some bandwidth, the unnecessary params are stripped
+	 * and standarized to match the curated content items
+	 *
+	 * @param $item
+	 * @return array
+	 */
+	private function processTrendingArticlesItem( $item ) {
+		global $wgArticlePath;
+
+		$processedItem = [];
+
+		if ( !empty( $item ) && is_array( $item ) ) {
+			$paramsMap = [
+				// params fetched from ArticlesApi => params passed to response
+				'title' => 'label',
```

Comment:
what is the real value of changing those keys?
Can't we just use what is provided?